data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -4209,6 +4209,30 @@
     "actionid": 294,
     "actioncode": "",
     "tenantId": "pg"
-   }
+   },
+   {
+     "rolecode": "PQM_TP_OPERATOR",
+     "actionid": 331,
+     "actioncode": "",
+     "tenantId": "pg"
+   },
+    {
+     "rolecode": "SUPERUSER",
```

Comment:
why role action for business service update ?
it is an interservice call right ?
 @Priyanka-eGov @sivajiganesh-egov